polyfill/lib/ecmascript.mjs

```diff
@@ -2960,7 +2960,24 @@ export const ES = ObjectAssign({}, ES2020, {
       µs2,
       ns2
     );
+
+    const timeSign = ES.DurationSign(0, 0, 0, deltaDays, hours, minutes, seconds, milliseconds, microseconds, nanoseconds);
     ({ year: y1, month: mon1, day: d1 } = ES.BalanceISODate(y1, mon1, d1 + deltaDays));
+    const dateSign = ES.CompareISODate(y1, mon1, d1, y2, mon2, d2);
+    if (dateSign === timeSign) {
```

Comment:
Using `===` here confused me, but I guess it's because the ComparisonResult (as in, C strcmp order) has the opposite sign from DurationSign? Maybe it would be clearer to `CompareISODate(y2, mon2, d2, y1, mon1, d1)` or something like that.